service/src/main/java/eu/europa/ec/fisheries/uvms/movement/service/entity/area/Movementarea.java

```diff
@@ -37,12 +37,14 @@ This file is part of the Integrated Fisheries Data Management (IFDM) Suite. The
 })
 @XmlRootElement
 @NamedQueries({
-    @NamedQuery(name = "Movementarea.findAll", query = "SELECT m FROM Movementarea m")})
+    @NamedQuery(name = "Movementarea.findAll", query = "SELECT m FROM Movementarea m"),
+    @NamedQuery(name = Movementarea.FIND_AREA_IDS_BY_AREA_REMOTEID_AND_NAME, query = "SELECT ma.movareaAreaId.areaId FROM Movementarea ma WHERE ma.movareaAreaId.remoteId IN :remoteIds AND ma.movareaAreaId.areaType.name IN :areaNames")})
```

Comment:
NO! This is wrong, it may bring areas of the wrong `remoteId`-`areaType.name` combination!